contents/scalability/scalable-development.md

```diff
@@ -0,0 +1,71 @@
+# Scalable Development - An Introduction
+
+Authors: Loh Jia Shun Kenneth
+
+## Scalable Development
+When Pokémon Go first launched in July 6, the heavy traffic from users caused its servers to crash. However, as of 2017, Google handles at least 3.5 billion searches daily, Whatsapp handles at least 10 billion messages a day, and Facebook hosts 1.2 billion daily active users. What makes these companies different?
+
+The key difference is their server infrastructure. A scalable server allows a company to provide reliable service even as the user base increases. Even if your service might not have to cater to millions of users at the moment, designing scalable software now will reduce the need to restructure the entire backend when the need arises.
```

Comment:
>The key difference is their server infrastructure.

Did you mean `The key difference is the scalability of their server infrastructure`?